### PR TITLE
add cta buttons to list view

### PR DESCRIPTION
**Pull request by trishasalas**
https://central.tri.be/issues/71092

_related_:
https://github.com/moderntribe/event-tickets-plus/pull/281
https://github.com/moderntribe/tribe-common/pull/334
https://github.com/moderntribe/event-tickets/pull/342

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:30](https://github.com/moderntribe/the-events-calendar/blob/0cc1ced5ae31c737aff5dc87525a411e57497e86/src/views/list/single-event.php#L30)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:51](https://github.com/moderntribe/the-events-calendar/blob/0cc1ced5ae31c737aff5dc87525a411e57497e86/src/views/list/single-event.php#L51)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:57](https://github.com/moderntribe/the-events-calendar/blob/0cc1ced5ae31c737aff5dc87525a411e57497e86/src/views/list/single-event.php#L57)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:60](https://github.com/moderntribe/the-events-calendar/blob/0cc1ced5ae31c737aff5dc87525a411e57497e86/src/views/list/single-event.php#L60)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:76](https://github.com/moderntribe/the-events-calendar/blob/0cc1ced5ae31c737aff5dc87525a411e57497e86/src/views/list/single-event.php#L76)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-featured.php:67](https://github.com/moderntribe/the-events-calendar/blob/0cc1ced5ae31c737aff5dc87525a411e57497e86/src/views/list/single-featured.php#L67)

> via jenkins/codesniffer-phpcs

File: src/views/list/single-event.php [hunk 1]
```diff
@@ -24,56 +24,57 @@
 
 ?>
 
-<!-- Event Cost -->
+	<!-- Event Cost -->
```
Comment (barryhughes) [inline src/views/list/single-event.php]: Tab back?

File: src/views/list/single-event.php [hunk 2]
```diff
@@ -24,56 +24,57 @@
...
 <?php if ( tribe_get_cost() ) : ?>
 	<div class="tribe-events-event-cost">
-		<span><?php echo tribe_get_cost( null, true ); ?></span>
+		<span class="ticket-cost"><?php echo tribe_get_cost( null, true ); ?></span>
+		<?php do_action( 'tribe_events_inside_cost' ) ?>
```
Comment (barryhughes) [inline src/views/list/single-event.php]: Document this new hook?

File: src/views/list/single-event.php [hunk 3]
```diff
@@ -24,56 +24,57 @@
...
 	</div>
 <?php endif; ?>
 
-<!-- Event Title -->
+	<!-- Event Title -->
```
Comment (barryhughes) [inline src/views/list/single-event.php]: Tab back?

File: src/views/list/single-featured.php [hunk 4]
```diff
@@ -68,6 +61,14 @@
 </div><!-- .tribe-events-event-meta -->
 <?php do_action( 'tribe_events_after_the_meta' ) ?>
 
+	<!-- Event Cost -->
+<?php if ( tribe_get_cost() ) : ?>
+	<div class="tribe-events-event-cost featured-event">
+		<span class="ticket-cost"><?php echo tribe_get_cost( null, true ); ?></span>
+		<?php do_action( 'tribe_events_inside_cost' ) ?>
```
Comment (barryhughes) [inline src/views/list/single-featured.php]: Document this new hook or note that it is documented elsewhere (and cite the file), if you do this from within another template that also uses the hook.

**Comment by barryhughes** [comment]
Remember to update or add the `@version` tag in any template files you modify.

File: src/views/list/single-featured.php (hunk 4 above)
Comment (trishasalas) [inline src/views/list/single-featured.php]: @jesseeproductions this one as well?

File: src/views/list/single-event.php (hunk 2 above)
Comment (trishasalas) [inline src/views/list/single-event.php]: @jesseeproductions can you do this?

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:30](https://github.com/moderntribe/the-events-calendar/blob/64d7a850724a0283907978e64973ea05beebcf2e/src/views/list/single-event.php#L30)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:51](https://github.com/moderntribe/the-events-calendar/blob/64d7a850724a0283907978e64973ea05beebcf2e/src/views/list/single-event.php#L51)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:57](https://github.com/moderntribe/the-events-calendar/blob/64d7a850724a0283907978e64973ea05beebcf2e/src/views/list/single-event.php#L57)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:60](https://github.com/moderntribe/the-events-calendar/blob/64d7a850724a0283907978e64973ea05beebcf2e/src/views/list/single-event.php#L60)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:76](https://github.com/moderntribe/the-events-calendar/blob/64d7a850724a0283907978e64973ea05beebcf2e/src/views/list/single-event.php#L76)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-featured.php:67](https://github.com/moderntribe/the-events-calendar/blob/64d7a850724a0283907978e64973ea05beebcf2e/src/views/list/single-featured.php#L67)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:30](https://github.com/moderntribe/the-events-calendar/blob/1e38a52eb4a31e4ee7110ccba3c3a2a9de414c73/src/views/list/single-event.php#L30)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:58](https://github.com/moderntribe/the-events-calendar/blob/1e38a52eb4a31e4ee7110ccba3c3a2a9de414c73/src/views/list/single-event.php#L58)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:64](https://github.com/moderntribe/the-events-calendar/blob/1e38a52eb4a31e4ee7110ccba3c3a2a9de414c73/src/views/list/single-event.php#L64)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:67](https://github.com/moderntribe/the-events-calendar/blob/1e38a52eb4a31e4ee7110ccba3c3a2a9de414c73/src/views/list/single-event.php#L67)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/1e38a52eb4a31e4ee7110ccba3c3a2a9de414c73/src/views/list/single-event.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-featured.php:67](https://github.com/moderntribe/the-events-calendar/blob/1e38a52eb4a31e4ee7110ccba3c3a2a9de414c73/src/views/list/single-featured.php#L67)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:30](https://github.com/moderntribe/the-events-calendar/blob/08eb510af45cbe83c636902024f0dfe4fe09f5df/src/views/list/single-event.php#L30)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:58](https://github.com/moderntribe/the-events-calendar/blob/08eb510af45cbe83c636902024f0dfe4fe09f5df/src/views/list/single-event.php#L58)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:64](https://github.com/moderntribe/the-events-calendar/blob/08eb510af45cbe83c636902024f0dfe4fe09f5df/src/views/list/single-event.php#L64)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:67](https://github.com/moderntribe/the-events-calendar/blob/08eb510af45cbe83c636902024f0dfe4fe09f5df/src/views/list/single-event.php#L67)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/08eb510af45cbe83c636902024f0dfe4fe09f5df/src/views/list/single-event.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-featured.php:67](https://github.com/moderntribe/the-events-calendar/blob/08eb510af45cbe83c636902024f0dfe4fe09f5df/src/views/list/single-featured.php#L67)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:30](https://github.com/moderntribe/the-events-calendar/blob/4fa5aaf00a67a085974e06465c694c4c894f4241/src/views/list/single-event.php#L30)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:58](https://github.com/moderntribe/the-events-calendar/blob/4fa5aaf00a67a085974e06465c694c4c894f4241/src/views/list/single-event.php#L58)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:64](https://github.com/moderntribe/the-events-calendar/blob/4fa5aaf00a67a085974e06465c694c4c894f4241/src/views/list/single-event.php#L64)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:67](https://github.com/moderntribe/the-events-calendar/blob/4fa5aaf00a67a085974e06465c694c4c894f4241/src/views/list/single-event.php#L67)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/4fa5aaf00a67a085974e06465c694c4c894f4241/src/views/list/single-event.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-featured.php:67](https://github.com/moderntribe/the-events-calendar/blob/4fa5aaf00a67a085974e06465c694c4c894f4241/src/views/list/single-featured.php#L67)

> via jenkins/codesniffer-phpcs

File: src/views/list/single-event.php [hunk 5]
```diff
@@ -24,56 +24,64 @@
...
 <?php do_action( 'tribe_events_before_the_event_title' ) ?>
-<h2 class="tribe-events-list-event-title">
-	<a class="tribe-event-url" href="<?php echo esc_url( tribe_get_event_link() ); ?>" title="<?php the_title_attribute() ?>" rel="bookmark">
-		<?php the_title() ?>
-	</a>
-</h2>
+	<h2 class="tribe-events-list-event-title">
```
Comment (borkweb) [inline src/views/list/single-event.php]: These extra tabs should be undone.

File: src/views/list/single-featured.php [hunk 6]
```diff
@@ -68,6 +61,17 @@
 </div><!-- .tribe-events-event-meta -->
 <?php do_action( 'tribe_events_after_the_meta' ) ?>
 
+	<!-- Event Cost -->
+<?php if ( tribe_get_cost() ) : ?>
+	<div class="tribe-events-event-cost featured-event">
+		<span class="ticket-cost"><?php echo tribe_get_cost( null, true ); ?></span>
```
Comment (borkweb) [inline src/views/list/single-featured.php]: `esc_html()` the `tribe_get_cost()`?

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:30](https://github.com/moderntribe/the-events-calendar/blob/b769a7f7b848ca2578905ac2c34132ec58789d45/src/views/list/single-event.php#L30)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:58](https://github.com/moderntribe/the-events-calendar/blob/b769a7f7b848ca2578905ac2c34132ec58789d45/src/views/list/single-event.php#L58)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:64](https://github.com/moderntribe/the-events-calendar/blob/b769a7f7b848ca2578905ac2c34132ec58789d45/src/views/list/single-event.php#L64)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:67](https://github.com/moderntribe/the-events-calendar/blob/b769a7f7b848ca2578905ac2c34132ec58789d45/src/views/list/single-event.php#L67)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/b769a7f7b848ca2578905ac2c34132ec58789d45/src/views/list/single-event.php#L83)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:30](https://github.com/moderntribe/the-events-calendar/blob/0f6a93e5d803b230788fede9f9e39785a0c74d24/src/views/list/single-event.php#L30)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:58](https://github.com/moderntribe/the-events-calendar/blob/0f6a93e5d803b230788fede9f9e39785a0c74d24/src/views/list/single-event.php#L58)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:64](https://github.com/moderntribe/the-events-calendar/blob/0f6a93e5d803b230788fede9f9e39785a0c74d24/src/views/list/single-event.php#L64)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:67](https://github.com/moderntribe/the-events-calendar/blob/0f6a93e5d803b230788fede9f9e39785a0c74d24/src/views/list/single-event.php#L67)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/0f6a93e5d803b230788fede9f9e39785a0c74d24/src/views/list/single-event.php#L83)

> via jenkins/codesniffer-phpcs

File: src/views/list/single-event.php [hunk 7]
```diff
@@ -27,7 +27,15 @@
...
 	<div class="tribe-events-event-cost">
-		<span><?php echo tribe_get_cost( null, true ); ?></span>
+		<span class="ticket-cost"><?php echo tribe_get_cost( null, true ); ?></span>
+		<?php
+			/**
+			 * Runs after cost is displayed in list style views
+			 *
+			 * @since 4.5
+			 */
+			do_action( 'tribe_events_inside_cost' )
```
Comment (borkweb) [inline src/views/list/single-event.php]: PHP should be level with the `<?php` tag like so:

```php
<?php
/**
 * laksdfjasldfjsaldf
 */
do_action( ... );
?>
```

File: src/views/list/single-featured.php [hunk 8]
```diff
@@ -68,6 +61,17 @@
 </div><!-- .tribe-events-event-meta -->
 <?php do_action( 'tribe_events_after_the_meta' ) ?>
 
+<!-- Event Cost -->
+<?php if ( tribe_get_cost() ) : ?>
+	<div class="tribe-events-event-cost featured-event">
+		<span class="ticket-cost"><?php echo esc_html( tribe_get_cost( null, true ) ); ?></span>
+		<?php
+			/** This action is documented in the-events-calendar/src/views/list/single-event.php */
+			do_action( 'tribe_events_inside_cost' )
```
Comment (borkweb) [inline src/views/list/single-featured.php]: Un-tab this (and the comment by one

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:30](https://github.com/moderntribe/the-events-calendar/blob/7ca334fde14b340df281ef11a231818aa0695f93/src/views/list/single-event.php#L30)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:58](https://github.com/moderntribe/the-events-calendar/blob/7ca334fde14b340df281ef11a231818aa0695f93/src/views/list/single-event.php#L58)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:64](https://github.com/moderntribe/the-events-calendar/blob/7ca334fde14b340df281ef11a231818aa0695f93/src/views/list/single-event.php#L64)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:67](https://github.com/moderntribe/the-events-calendar/blob/7ca334fde14b340df281ef11a231818aa0695f93/src/views/list/single-event.php#L67)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/7ca334fde14b340df281ef11a231818aa0695f93/src/views/list/single-event.php#L83)

> via jenkins/codesniffer-phpcs

**Pull request merged**